sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DoNotUseRandom.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Security.Cryptography;
+
+namespace Tests.Diagnostics
+{
+    class Program
+    {
+        public void Main()
```

Comment:
Please add tests with fields, properties and local functions.

Reply:
Comment still valid.

Reply:
irrelevant - we raise when we see ctor